docs/tutorial.md

```diff
@@ -225,3 +226,80 @@ Check out the example app.
 Another recommended solution would be to implement your own custom keyboard on your screen directly.
 It allows you to embed it in your page directly next to the text field (as many famous TV apps do).
 No example yet!
+
+## Handle remote pointers
+
+Remote pointers represents the movement-based cursor some TV have (LG and the Magic Remote for instance). To handle these movements, an API is available in the library.
+Here are the key elements which can handle remote pointers :
+
+- `SpatialNavigationDeviceTypeProvider`
+- `SpatialNavigationFocusableView`
+- `SpatialNavigationScrollView`
+- `SpatialNavigationVirtualizedList`
+- `SpatialNavigationVirtualizedGrid`
+
+### Handling basic pointer hovering
+
+In order to know what device the user is actively using, your app needs to have access to the `DeviceContext`. We suggest you place it a the root of your app :
+
+```tsx
+function App(): JSX.Element {
+  return (
+    <SpatialNavigationDeviceTypeProvider>
+      <DefaultFocus>
+        <SomeFocusableComponent onSelect={() => console.log('selected first element')} />
+      </DefaultFocus>
+      <SomeFocusableComponent onSelect={() => console.log('selected second element')} />
+    </SpatialNavigationDeviceTypeProvider>
+  );
+}
+
+const SomeFocusableComponent = ({ onSelect }: Props) => {
+  return (
+    <SpatialNavigationFocusableView onSelect={onSelect}>
```

Comment:
You should insist that the SpatialNavigationFocusableView is key for the pointers to work 😉

Reply:
We should make it clear in our examples, too. Can you search for `isFocusable` in the docs and make sure that everytime we use a node that is focusable, we now use the FocusableView instead?

Reply:
Yes, great idea !

Reply:
Done